python/ray/dataframe/dataframe.py

```diff
@@ -192,6 +201,15 @@ def _map_partitions(self, func, index=None):
 
         return DataFrame(new_df, self.columns, index=index)
 
+    def _update_inplace(self, new_dfs):
+        """Updates the current DataFrame inplace
+        """
+        assert(len(new_dfs) > 0)
```

Comment:
DataFrames can be empty. This is fine to leave for now, but we need to handle this case (also above in the constructor)